src/App.js

```diff
@@ -1,10 +1,18 @@
+import { Routes, Route } from 'react-router-dom';
 import './App.scss';
+import NavBar from './components/nav-bar/NavBar';
+import AuthorsPage from './pages/AuthorsPage';
+import Home from './pages/Home';
 
 function App() {
   return (
     <div className="App">
       <header className="App-header">
-        <h1>Hello World</h1>
+        <NavBar />
+        <Routes>
+          <Route path="/" element={<Home />} />
+          <Route path="/authors" element={<AuthorsPage />} />
```

Comment:
In order to respect the logic that this project follows, we need to respect the milestone requirements regarding the names of components and the name of the different sections of our app.

- [ ] Kindly, change the path name from AuthorsPage to Categories;

Screenshot from project requirements:
![image](https://user-images.githubusercontent.com/83596135/165865610-5f8a361c-9562-4085-9974-2e4b2c994e8c.png)

Reply:
@nicupop729 thank you for your feedback. I will change it per requirements ASAP 🚀